federator/package.json

```diff
@@ -12,11 +12,16 @@
   ],
   "author": "IOVLabs",
   "scripts": {
-    "start": "node ./src/main.js",
+    "start": "node ./built/federator/src/main.js",
+    "start-js": "node ./src/main.js",
```

Comment:
what is the difference between this and start? (`-js` doesn't tell me much)

Reply:
The `start-js` script uses directly the files before being compiled... works just with plain java script, after we created and write code in typescript is going to be need to compile before starting

Reply:
makes sense, thanks!